app/src/main/res/layout/fragment_safety.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout android:id="@+id/safety_mode_container"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:visibility="visible"
+    tools:visibility="visible"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <ImageView
+        android:id="@+id/back_safety"
+        android:layout_gravity="top|start"
+        android:src="@drawable/ic_back"
+        style="@style/BackButtonStyle" />
+
+    <TextView
```

Comment:
nit: how about styles for this textViews?
you've already create for `BackButtonStyle`

Reply:
For now, I've added style for the same view (BackButton) in different fragments. 
Seems other views are used in one place

Reply:
okay